source/summary.md

```diff
@@ -0,0 +1,373 @@
+    <li>Meaningful elements can be distinguished [<a href="https://bokeh-a11y-audit.readthedocs.io/plotting-interface/meaningful-elements.html" aria-label="Meaningful Elements evidence in plotting interface" title="Meaningful Elements evidence in plotting interface">PI-15</a>, <a href="https://bokeh-a11y-audit.readthedocs.io/annotations/meaningful-elements.html" aria-label="Meaningful Elements evidence in annotations" title="Meaningful Elements evidence in annotations">A-12</a>].</li>
+</ul>
+
+##### Example
+
+<p><i>Example taken from annotations [<a href="" aria-label="Contrast evidence in annotation" title="Contrast evidence in annotation">23</a>].</i> Below demonstrates the near-invisibility of the interactive annotation element used to toggle a line chart on and off.</p>
+
+```{figure} ./annotations/assets/annotations_contrast_1.png
+:width: 100%
+:alt: A double line chart is shown. A color selection dropper is highlighting a selected 'muted', grayed out line. The contrast checking score of 1.27 is shown on the bottom left corner (fails).
+
+A double line chart is shown. A color selection dropper is highlighting a selected 'muted', grayed out line. The contrast checking score of 1.27 is shown on the bottom left corner (fails).
+```
+
+#### <a id="finding-2B" href="#finding-2B" aria-label="Finding 2B"><span aria-hidden="true">#</span></a> B. Understanding is not easy
+
+In general, everything should be easy to understand for folks who have never before encountered a Bokeh chart, its interactive capabilities, and its annotations.
+
+_What_ a chart shows should be described. _Why_ some information has been visualized should be explained. _How_ a user can interact should be clearly explained, including with instructions provided for more complex interactions (like lasso, etc). All text and labels should be made legible and presented at a reasonable reading level.
+
+Bokeh's cognitive support overall seems to be geared toward supporting software developers who are using the library while generally leaving end-user understandability up to the user's own ability to poke around with interactivity, grok context/use, and self-interpret.
+
+In general, Bokeh doesn't appear to provide _any_ assistance to developers in authoring textual descriptions, providing end-user oriented instructions/explanations, and also appears to have minimal support for automatically simplifying/formatting data-oriented numerical values (such as using too many decimal points of precision in a label).
```

Comment:
Seeing this called out is great, as I have come to realise this is a too-often ignored part of making visualisations more accessible and understandable.